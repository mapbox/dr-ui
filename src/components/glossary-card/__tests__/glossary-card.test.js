import renderer from 'react-test-renderer';
import { testCases } from './glossary-card-test-cases.js';

describe('glossary-card', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
