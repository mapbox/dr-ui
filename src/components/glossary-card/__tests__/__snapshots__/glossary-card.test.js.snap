// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`glossary-card Basic renders as expected 1`] = `
<div
  className="mb6 w240 flex-child"
>
  <a
    className="color-gray-dark transition shadow-darken10-on-hover round clip inline-block w-full px12 py12 unprose"
    href="#donut"
    style={
      Object {
        "verticalAlign": "top",
      }
    }
  >
    <div
      className="px6 pb6"
    >
      <div
        className="mb6 txt-l"
      >
        Donut
      </div>
      <div
        className="txt-s color-gray"
      >
        A delicious fried dough treat typically shaped as a circle with a hole in the middle.
      </div>
    </div>
  </a>
</div>
`;
